src/stories/tokens-animation.stories.mdx

```diff
@@ -0,0 +1,6 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import { DesignTokenDocBlock } from 'storybook-design-token/dist/doc-blocks';
+
+<Meta title="Design Tokens/Animation" />
```

Comment:
I believe those are included by default and bind automatically where supported now:
https://storybook.js.org/docs/react/essentials/controls

Reply:
Can still be configured custom, but probably a pattern to test when we get to the components phase.

Reply:
What do you think of this pattern?
```
import { Story, Meta } from '@storybook/react/types-6-0';

import { Example, ExampleProps } from './index';

export default {
  title: 'Example',
  component: Example,
  args: {
    children: 'An example',
  },
} as Meta;

const Template: Story<ExampleProps> = (args: any) => {
  return <Example {...args} />;
};

export const Basic = Template.bind({});

```

Reply:
No complaints here.

Reply:
ok - wasn't seeing much movement yet on the Figma design system proposals, so thought it might just be good to add an example story for the Button component.

I don't have a ton of recent React experience, but I followed the pattern above as best I could minus the Typescript aspects which I don't think are established in this repo yet.

Let me know if you want anything tweaked.